setup doc page for morphological operators and more (#76)
JuliaImages/ImageMorphology.jl: README.md

[![][action-img]][action-url]
[![][pkgeval-img]][pkgeval-url]
[![][codecov-img]][codecov-url]
[![][docs-stable-img]][docs-stable-url]
[![][docs-dev-img]][docs-dev-url]

> This package provides morphology-related functionalities for structure analysis and image processing.

Reviewer: ```suggestion
This package provides morphology operations for structure analysis and image processing.
```

> ImageMorphology is a sub-package of the umbrella package [Images.jl][images-url] -- you can choose
> to use either `using Images` or `using ImageMorphology` for the functionality.

Reviewer: ```suggestion
ImageMorphology is a sub-package of the umbrella package [Images.jl][images-url] -- either `using Images` or `using ImageMorphology` will give you access to this functionality.
```

## Documentation

Full documentation and description can be found [here][docs-stable-url].

## Installation

> Just like all normal Julia packages, you can use Pkg to install it:

Reviewer: ```suggestion
Just like all normal Julia packages, you can use [Pkg](https://pkgdocs.julialang.org/v1/getting-started/) to install it:
```
